lib/features/BentaSignUp/SignUpPage1.dart

```diff
@@ -0,0 +1,146 @@
+        padding: const EdgeInsets.all(45.0),
+        child:
+        SingleChildScrollView(
+          physics: const BouncingScrollPhysics(),
+          child: Form(
+            key: _formKey,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+
+                Center(
+                  child: Container(
+                    width: 100,
+                    height: 100,
+                    child: Image.asset(
+                      'assets/images/benta_logo.png',
+                      fit: BoxFit.cover,
+                    ),
+                  ),
+                ),
+
+                const SizedBox(height: 30),
+
+                const Text(
+                  'Sign-up',
+                  style: TextStyle(
+                      fontSize: 44,
+                      fontWeight: FontWeight.bold,
+                      color: Colors.white,
+                  ),
+                ),
+
+                const Text(
+                  'Step 1',
+                  style: TextStyle(fontSize: 28, fontWeight: FontWeight.bold, color: Colors.white),
+                ),
+
+                const SizedBox(height: 40),
+
+                const Text(
+                  'What is the name of your business?',
+                  style: TextStyle(fontSize: 20, color: Colors.white),
+                ),
+
+                const SizedBox(height: 12),
+
+                // TextFormField for Business Name
+                TextFormField(
+                  decoration: InputDecoration(
+                    filled: true,
+                    fillColor: Colors.grey[200],
+                    border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(25.0),
+                    ),
+                  ),
+                  validator: (value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Please enter the name of your business';
+                    }
+                    return null;
+                  },
+                ),
+
+                const SizedBox(height: 40),
+
+                const Text(
+                  'What is your user name?',
+                  style: TextStyle(fontSize: 20, color: Colors.white),
+                ),
+
+                const SizedBox(height: 12),
+                // TextFormField for Password
+                TextFormField(
+                  decoration: InputDecoration(
+                    filled: true,
+                    fillColor: Colors.grey[200],
+                    border: OutlineInputBorder(
+                      borderRadius: BorderRadius.circular(25.0),
+                    ),
+                  ),
+                  validator: (value){
+                    if (value == null || value.isEmpty) {
+                      return 'Please enter your user name';
+                    } else if (value.length < 5) {
+                      return 'Username must be at least 5 characters long';
```

Comment:
I recommend to set usernames by at least 1 character. There may be some people who want to be named three or two letters only, depending on their preference.